test/idl/consistency.js

```diff
@@ -40,6 +39,7 @@ const ignorableAnomalies = [
 
 function writeAnomalies(report) {
   function writeAnomaly(anomaly) {
+    return anomaly.content;
     return `- ${anomaly.name} in ${anomaly.specs.map(s => s.shortname).join(', ')}: ${anomaly.message}`;
```

Comment:
```suggestion
```
(or get rid of `writeAnomaly` altogether since it can reasonably be inlined in L47 now)

Reply:
Oh thanks, forgot to clean the code!